Trackovid19-web/package-lock.json

```diff
@@ -5425,8 +5425,7 @@
         "ansi-regex": {
           "version": "2.1.1",
           "resolved": false,
-          "integrity": "sha1-w7M6te42DYbg5ijwRorn7yfWVN8=",
```

Comment:
Can you remove the updates on this file? You didn't updated the package.json, so you should not add this.